lerna.json

```diff
@@ -1,13 +1,11 @@
 {
-  "lerna": "2.0.0-rc.1",
   "packages": [
-    "packages/*",
-    "."
+    "packages/*"
   ],
   "hoist": true,
   "nohoist": "guppy-pre-commit",
   "version": "independent",
-  "commands": {
+  "command": {
```

Comment:
Is bootstrap's concurrency still configurable? I think that this option doesn't exist any more.

Reply:
![image](https://user-images.githubusercontent.com/2270764/46344169-88cad180-c640-11e8-8bdc-ca798a3b521a.png)